backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -43,14 +43,45 @@
   }
 }
 
+// Page Layout
+//---------------------------------------------------
+
+html {
+  height: 100%;
+}
+
+body {
+  position: relative;
+  min-height: 100%;
+}
+
+.admin-nav {
+  @include position(fixed, 0 null 0 0);
```

Comment:
You beat me to it again @tvdeyen I was just about to add that very feature (js affix). I feel bad that we didn't foresee this problem sooner, I guess that's a drawback of developing on ginormous monitors.

I would love to have been able to fix the problem with CSS only, but adding any type of scroll (even to only 1 direction `overflow-x: auto`) cuts off every overflow. So at the very least we would need to position the flyouts with javascript (append to body, position with js). But if we're going to do JS we might as well take the wordpress route and have the entire nav follow you around.